### PR TITLE
Update changelog for 0.7.1 release.

### PR DESCRIPTION
**Pull request by mithrandi**

### CODE REVIEW

**Comment by mithrandi** [comment]
You can peruse https://github.com/twisted/epsilon/compare/0.7.0...releases/0.7.1 to see all of the revisions since the last release.

File: NEWS.rst [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    unrelated *AMP* communications over the same connection.
+
+0.5.10 (2008-08-12):
+  - Added the ``epsilon.caseless`` module, with case-insensitive string
+    wrappers.
+  - Better ``repr()`` for ``epsilon.structlike.record`` added.
+  - ``epsilon.juice`` now uses ``twisted.internet.ssl`` instead of
+    ``epsilon.sslverify``.
+
+0.5.9 (2008-01-18):
```
Comment (jsza) [inline NEWS.rst]: It seems that because no lines follow this one it isn't shown as bold.

Comment (mithrandi) [inline NEWS.rst]: Fixed.

File: NEWS.rst [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  - ``extime.Time`` now avoids ``time_t`` overflow bugs.
+
+0.5.1 (2006-06-22):
+  - Added hotfix for ``twisted.test.proto_helpers.StringTransport``.
+
+0.5.0 (2006-06-12):
+  - Replaced ``'%y'`` with ``'%Y'`` in ``Time.asHumanly`` output - the year is
+    now four digits, rather than two.
+  - Added new ``epsilon.structlike`` functionality for simple record.
+  - All uses of defer.wait and deferredResult were removed from the tests.
```
Comment (jsza) [inline NEWS.rst]: Based on other examples, it seems like `defer.wait` and `deferredResult` should be wrapped in code markup here.

File: NEWS.rst (hunk 1 above)
Comment (mithrandi) [inline NEWS.rst]: Fixed.

File: NEWS.rst [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+  - Added ``epsilon.juice``, an asynchronous messaging protocol slated for
+    inclusion in *Twisted*.  Improved a few features, such as the ``repr`` of
+    JuiceBox instances.  This was moved from *Vertex*.
```
Comment (jsza) [inline NEWS.rst]: Should JuiceBox be italicized here?

File: NEWS.rst [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+0.3.0 (2005-11-02):
+  - Added ``SchedulingService``, an ``IService`` implementation, to
+    ``epsilon.cooperator``.
+  - Added ``autosetup``, a utility to actually include files in *distutils*
+    releases, to ``epsilon.setuphelper``.
+
+0.2.1 (2005-10-25):
+  - Added ``short`` to ``epsilon.versions.Version``.
+  - fixed *setup.py* to use ``epsilon.version.short`` rather than static
```
Comment (jsza) [inline NEWS.rst]: The casing seems to be incorrect at the start of this line. However, it was the same prior to this branch.

File: NEWS.rst (hunk 2 above)
Comment (mithrandi) [inline NEWS.rst]: Fixed.

File: NEWS.rst (hunk 3 above)
Comment (mithrandi) [inline NEWS.rst]: Added code markup instead.

File: NEWS.rst (hunk 4 above)
Comment (mithrandi) [inline NEWS.rst]: Fixed case.

**Comment by jsza** [comment]
Looks good to merge.

**Pull request merged**